### PR TITLE
add parser to types

### PR DESCRIPTION
**Pull request by ekoeryanto**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
could you add a test for the new type? Thanks!

**Comment by mcollina** [comment]
cc @fastify/typescript

File: formbody.d.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { FastifyPlugin } from 'fastify'
 
 export interface FormBodyPluginOptions {
   bodyLimit?: number
+  parser?: (str: string) => string
```
Comment (L2jLiga) [inline formbody.d.ts:5]: I think return type is incorrect, for example default parser returns dictionary.
I would consider to change it to unknown

Comment (rodrigoscna) [reply formbody.d.ts:5]: How about returning `Record<string, unknown>`?

Comment (L2jLiga) [reply formbody.d.ts:5]: Both looks applicable for me, not sure what is the best approach

**Pull request closed without merging**